bokehjs/src/lib/core/visuals/visual.ts

```diff
@@ -37,6 +37,12 @@ export abstract class VisualProperties {
   abstract get doit(): boolean
 
   update(): void {}
+
+  protected _get_css_value(name: string): string {
+    const style = getComputedStyle(this.obj.el)
+    const prefix = this.prefix.replaceAll("_", "-")
```

Comment:
visuals are consulted on every re-draw would it make a difference to pre-compute `self.css_prefix` once in the constructor?

Reply:
It would be good to do it, but the cost of this compared to the surrounding code is negligible.